modules/ztp-deploying-a-site.adoc

```diff
@@ -182,6 +182,6 @@ The `SiteConfig` CR creates the following CRs on the hub cluster:
 * `ClusterDeployment`
 * `InfraEnv`
 * `NMStateConfig` - One per node
-* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, chronyd, mountpoint hiding, sctp enablement, and more.
+* `ExtraManifestsConfigMap` - Extra manifests. The additional manifests include workload partitioning, mountpoint hiding, sctp enablement, and more.
```

Comment:
Can we add something like "They are automatically merged into a single manifest per role named as predefined-extra-manifests-<role>."?

Reply:
@tliu2021 I updated this part, could you please take a look?